src/utils/processUpload.js

```diff
@@ -5,7 +5,7 @@ import loadAndCompressIfNecessary from './loadAndCompressIfNecessary';
 import { createSample, updateSampleFile } from '../redux/actions/samples';
 import UploadStatus from './UploadStatus';
 
-const compressAndUpload = async (sample, activeProjectUuid, dispatch) => {
+const compressAndUpload = (sample, activeProjectUuid, dispatch) => {
```

Comment:
Why do you change this to sync?

Reply:
Because there's no need for an async when doing when handling promises with `.then()` and `.catch()` instead of with `await`.